test/suites/basic.sh

```diff
@@ -337,12 +337,12 @@ test_basic_usage() {
   # Create and start a container
   lxc launch testimage foo
   lxc list | grep foo | grep RUNNING
-  lxc stop foo --force  # stop is hanging
+  lxc stop foo --force  # stop is blocking
```

Comment:
I'd just drop the comment entirely as it's a common thing throughout the testsuite.